multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/UploadAppStep.java

```diff
@@ -94,6 +99,34 @@ public StepPhase executeAsyncStep(ProcessContext context) throws FileStorageExce
         return StepPhase.DONE;
     }
 
+    private CloudPackage createDockerPackage(CloudControllerClient client, CloudApplicationExtended application) {
+        UUID applicationGuid = client.getApplicationGuid(application.getName());
```

Comment:
Why isn't the guid in the `CloudApplicationExtended` object used but a request to the CC is made to fetch the guid?

Reply:
Because it set in BuildApplicationDeployModelStep and its guid is unknown -> the id is null.